src/ReactARTFiber.js

```diff
@@ -489,6 +485,10 @@ const ARTRenderer = ReactFiberReconciler({
     // Noop
   },
 
+  getChildHostContext() {
+    return null;
+  },
+
```

Comment:
Why the arbitrary relocation? Methods were alpha-sorted before (for easier readability).